src/Cimpress.Nancy.Services/ContentTypeVersionModuleExtender.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Cimpress.Nancy.Components;
+using System.Collections.Generic;
+using System.Linq;
+using Nancy;
+
+namespace Cimpress.Nancy.Services
+{
+    public class ContentTypeVersionModuleExtender : IVersionModuleExtender
+    {
+        private readonly IEnumerable<string> _entityMethods = new List<string> {"put", "post", "patch"};
+
+        public void After(NancyContext context)
+        {
+
+        }
+
+        public Response Before(NancyContext context)
+        {
+            var methodType = context.Request.Method.ToLower();
+            if (_entityMethods.Contains(methodType))
+            {
+                return new Response
+                {
+                    StatusCode = HttpStatusCode.BadRequest,
+                    ReasonPhrase = "Content-Type Header not supplied"
```

Comment:
Other things that might be more service oriented, in case people want to use the functionality of the base packages to add logging, error handling etc without enforcing service restrictions. If this is more tailored to the Cimpress/VIstaprint use case then that's fine (maybe all this package does is include the other Cimpress.Nancy.* libraries?)